giantswarm-plugin.yaml

```diff
@@ -9,6 +9,26 @@ spec:
     - name: TC_KUBECONFIG
     - name: CP_KUBECONFIG
     - name: CLUSTER_ID
+    - name: AZURE_CLIENT_ID
```

Comment:
didn't we get this data from the secret?

Reply:
Oh! VVery good point. I'll zap it!

Reply:
are you sure "zap" is an accepted word for our new code of conduct? :P

Reply:
With the claimed meaning of _to get rid of as if with sudden force_, in this context it must be :joy: